Mathlib/Topology/UnitInterval.lean

```diff
@@ -123,8 +123,9 @@ theorem symm_symm (x : I) : σ (σ x) = x :=
   Subtype.ext <| by simp [symm]
 #align unit_interval.symm_symm unitInterval.symm_symm
 
-theorem symm_bijective : Function.Bijective (symm : I → I) :=
-  Function.bijective_iff_has_inverse.mpr ⟨_, symm_symm, symm_symm⟩
+theorem symm_involutive : Function.Involutive (symm : I → I) := symm_symm
+
+theorem symm_bijective : Function.Bijective (symm : I → I) := symm_involutive.bijective
```

Comment:
Do you mind adding deprecated aliases for old names (`involutive_symm` and `bijective_symm` below)? Otherwise LGTM. Thanks!
bors d+

Reply:
Done

bors merge